sympy/stats/joint_rv_types.py

```diff
@@ -394,6 +395,120 @@ def MultivariateEwens(syms, n, theta):
     """
     return multivariate_rv(MultivariateEwensDistribution, syms, n, theta)
 
+#-------------------------------------------------------------------------------
+# Generalized Multivariate Log Gamma distribution ---------------------------------------------------------
+
+class GeneralizedMultivariateLogGammaDistribution(JointDistribution):
+
+    _argnames = ['delta', 'v', 'l', 'mu']
+    is_Continuous=True
+
+    def check(self, delta, v, l, mu):
+        _value_check((delta >= 0, delta <= 1), "delta must be in range [0, 1].")
+        _value_check((v > 0), "v must be positive")
+        for lk in l:
+            _value_check((lk > 0), "lamda must be a positive vector.")
+        for muk in mu:
+            _value_check((muk > 0), "mu must be a positive vector.")
+        _value_check(len(l) > 1,"the distribution should have at least"
+                                " two random variables.")
+
+    @property
+    def set(self):
+        from sympy.sets.sets import Interval
+        return S.Reals**len(self.l)
+
+    def pdf(self, *y):
+        from sympy.functions.special.gamma_functions import gamma
+        d, v, l, mu = self.delta, self.v, self.l, self.mu
+        n = Symbol('n', negative=False, integer=True)
+        k = len(l)
+        sterm1 = Pow((1 - d), n)/\
+                ((gamma(v + n)**(k - 1))*gamma(v)*gamma(n + 1))
+        sterm2 = Mul.fromiter([mui*li**(-v - n) for mui, li in zip(mu, l)])
+        term1 = sterm1 * sterm2
+        sterm3 = (v + n) * sum([mui * yi for mui, yi in zip(mu, y)])
+        sterm4 = sum([exp(mui * yi)/li for (mui, yi, li) in zip(mu, y, l)])
+        term2 = exp(sterm3 - sterm4)
+        return Pow(d, v) * Sum(term1 * term2, (n, 0, S.Infinity))
+
+def GeneralizedMultivariateLogGamma(syms, v, l, mu, omega_delta=None):
```

Comment:
what about `omega=None, delta=None`?

Then check that one and only one is defined.